javascript/webpacker_react-npm-module/dist/package.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
I'm not sure I understand the need to introduce this file in addition to `../package.json`?